### ISSUE TITLE
Mordino critters do not check for made man status properly when the player tampers with some containers

### ISSUE THREAD

**Issue by Question2** [BGforgeNet/Fallout2_Unofficial_Patch#63]
Mordino containers seem to use one of three scripts. This is NIMOR1LK.ssl : 

```
/*
	Copyright 1998-2003 Interplay Entertainment Corp.  All rights reserved.
*/

#define EXTRA_HEADER                "../headers/newreno.h"

#define WATCHER_OBJ                 new_reno_mordino_men_obj
#define DO_CHECK_TAMPER_CONDITION   (dude_family != family_mordino)

#include "../generic/zilocker.ssl"

```

There is a tamper check here, if the player is a mordino made man, they wont care if you try to use the container to take stuff.

The dev who did these appears to have forgotten to include the same tamper check for NIMOR2LK and NIMOR3LK however. This leads to the rather strange result where mordino critters go hostile if the player attempts to open some containers, even if they are already a mordino made man.

To fix, you just need to copy and paste #define DO_CHECK_TAMPER_CONDITION   (dude_family != family_mordino) into the other two scripts.

NIMYGDLK is also missing the check (its for the locked locker infront of Myron's room). So is NIBIGJLK for the locked desk in Big Jesus's room.

**Comment by burner1024** [comment]
1. It rather seems to me that it's intentional (and the reason to have multiple scripts in the first place):
  ```
  niMor1Lk.int    ; New Reno Mordino No Lock/No Ignore "Locker"   # local_vars=1
  niMor1Lk.int    ; New Reno Mordino FAMILY IGNORE "Locker"       # local_vars=1
  niMor2Lk.int    ; New Reno Mordino NO IGNORE "Locker"           # local_vars=1
  niMor3Lk.int    ; New Reno Mordino locker, no warning           # local_vars=1
  ```
2. I wonder if 1 and 2 are reversed, however, as it would make more sense to forbid looting the boss room rather than the others. @Question2 try checking other bosses rooms and see how it's done there.
3. I can't find where 3 is used at all.
4. For some reason, there's 2 entries for `niMor1Lk` in `scripts.lst`. @novarain, would you know if that has any effect at all?

To summarize, 2) may be actionable if there's evidence that they indeed confused the scripts, but otherwise I see no reason to change anything.
